src/azul/terra.py

```diff
@@ -239,6 +250,14 @@ class TDRClient(SAMClient):
     A client for the Broad Institute's Terra Data Repository aka "Jade".
     """
 
+    @property
+    def credential_scopes(self):
+        return [
+            *super().credential_scopes,
+            'https://www.googleapis.com/auth/devstorage.read_only',
+            'https://www.googleapis.com/auth/bigquery.readonly'
```

Comment:
You still need to change the title of the commit that makes these changes. I've left several comments about this already, but I suspect you've overlooked them.

Reply:
Ack, sorry, thanks for being patient.